scss/_back-to-top.scss

```diff
@@ -14,12 +14,13 @@
   --#{$prefix}back-to-top-zindex: #{$zindex-back-to-top};
   --#{$prefix}back-to-top-link-offset-top: #{$back-to-top-link-offset-top};
   --#{$prefix}back-to-top-link-icon: #{$back-to-top-icon};
-  --#{$prefix}back-to-top-link-icon-margin-left: #{$spacer * .5};
+  --#{$prefix}back-to-top-link-icon-margin-left: #{$back-to-top-icon-margin-left};
```

Comment:
I've extracted this modification into https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2340 targetting the _main_ branch to lighten the final review.